tests/test_remote_resource_logs.py

```diff
@@ -0,0 +1,169 @@
+import os
+import unittest
+from unittest import mock
+from unittest.mock import Mock
+
+from flow360.component.resource_base import (
+    Flow360Resource,
+    Position,
+    RemoteResourceLogs,
+)
+from flow360.log import LogLevel
+
+
+class TestRemoteResourceLogs(unittest.TestCase):
```

Comment:
can we stick to one python tests framework? Can we use pytest instead of unittest?